src/forms/FormLines.tsx

```diff
@@ -27,7 +27,8 @@ export interface FormLinesProps {
  */
 export function FormLines(props: FormLinesProps) {
   const settings = useContext(FormContext);
-  const { children, width = "md", labelSuffix = settings.labelSuffix } = props;
+  const { children, width = "full", labelSuffix = settings.labelSuffix } = props;
```

Comment:
I'm also changing the default form width to "full" so it just uses whatever the parent width is... Lately I've been in a lot of modals, and full is a better default for that...